m4/ac_check_mpc.m4

```diff
@@ -11,27 +11,14 @@ MPC=unknown
 MPC_WITH=""
 MPC_DEPEND=""
 
-MPC_MAKELIB=`printf 'mpc: $(MPCLIB).tar.gz %s  \\
-	mkdir -p $(EXTERN)/include $(EXTERN)/lib  \\
-	if ! test -r $(EXTERN)/include/mpc.h; then \\
-	    rm -rf $(MPCLIB) && \\
-	    tar -x -f $(MPCLIB).tar.gz -z -C extern && \\
-	    cd $(MPCLIB) && \\
-	    ./configure %s %s --prefix=$(EXTERN) && \\
-	    $(MAKE) install; \\
-	fi\n' "$MPFR_DEPEND" "$GMP_WITH" "$MPFR_WITH"`
```

Comment:
So this contains code for building `mpc` -- but `MPC_MAKELIB` is then never ever used, i.e. this code I removed here already is dormant in the current float `master` branch...